docs/guides/automations.md

```diff
@@ -0,0 +1,347 @@
+---
+description: Prefect walkthrough on how to use automations and common best practices 
+tags:
+    - automations
+    - event-driven
+    - trigger
+title: Automations and Common Use Cases
+search:
+  boost: 2
+---
+
+# Using Automations for Dynamic Responses
+
+From the Automations tutorial, we were able to see the capabilities of what an automation can do and how to configure them within the UI. 
+
+In this guide, we will showcase common usecases where automations can shine when responding to your workflows. First we will create a simple notification automation in just a few UI clicks. Then build upon that with an event based automation where a deployment will be kicked off. Lastly, we will combine these ideas to create a well alerted and responsive deployment pattern. 
+
+!!! Warning "Available only on Prefect Cloud"
```

Comment:
this is the wrong formatting

![image](https://github.com/PrefectHQ/prefect/assets/26799928/59017e10-8efd-4fea-9081-d2766e463e6d)